nistats/contrasts.py

```diff
@@ -98,9 +99,10 @@ def _fixed_effect_contrast(labels, results, con_vals, contrast_type=None):
             contrast = contrast_
         else:
             contrast = contrast + contrast_
+        n_contrasts += 1
```

Comment:
Hey @jeromedockes 
```python
    for n_contrasts, (lab, res, con_val) in enumerate(zip(labels, results, con_vals)):
```

Will work just as well, so no separate initialization and incrementation necessary.

Reply:
https://github.com/nistats/nistats/pull/357#discussion_r292681092